edits to Ch3 authorreview.Rmd (#114)
ropensci-org/blog-guidance: authorreview.Rmd

# Responding to Editor's Review {#respond}

```{block, type='summaryblock', echo=TRUE}
This chapter outlines how an author of a blog post or tech note can find and respond to comments from an editor in the GitHub pull request review interface.
```

> Once you have [submitted](#submitpost) your blog post or tech note, an editor will review your post providing comments on both [content](#content) (does it make sense?) and [technical](#technical) (does it follow our Style Guide?) aspects.

Reviewer: "does it make sense?" sounds a bit harsh. "Is your story told in an optimal way?" or so?

> Once you have [submitted](#submitpost) your blog post or tech note, an editor will review your post providing comments on both [content](#content) (does it make sense?) and [technical](#technical) (does it follow our Style Guide?) aspects.

Author: ```suggestion
Once you have [submitted](#submitpost) your blog post or tech note, an editor will review your post providing comments on both [content](#content) (is the narrative clear?) and [technical](#technical) (does it follow our Style Guide?) aspects.
```

> Browse [Chapter 5.1 Review a post](#review) to see how editors use the well documented [GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) to suggest edits, make comments, and approve or request changes to your pull request.

Reviewer: I'd actually drop "well documented" because it sounds a bit RTFM. :wink:

> Browse [Chapter 5.1 Review a post](#review) to see how editors use the well documented [GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) to suggest edits, make comments, and approve or request changes to your pull request.

Author: ```suggestion
Browse [Chapter 5.1 Review a post](#review) to see how editors use the [GitHub web interface](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/reviewing-changes-in-pull-requests) to suggest edits, make comments, and approve or request changes to your pull request.
```

To respond to an editor's review:

- Respond to every inline comment with a thumbs up or thumbs down emoji so the editor knows you have seen it.
- It's ok to explain why you disagree with some comments.

> - [_Resolve conversation_](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#resolving-conversations) for items you have addressed.

Reviewer: Question: why would I both resolve and add a thumbs up/down?

Author: believe it or not, people resolve conversations without making suggested changes and then indicate they did not see them

Reviewer: Ah ok!
